scripts/winesapos-install.sh

```diff
@@ -795,6 +809,12 @@ elif [[ "${WINESAPOS_DE}" == "plasma" ]]; then
     chroot ${WINESAPOS_INSTALL_DIR} crudini --ini-options=nospace --set /etc/xdg/konsolerc "Desktop Entry" DefaultProfile Vapor.profile
     # Image gallery and text editor.
     pacman_install_chroot gwenview kate
+elif [[ "${WINESAPOS_DE}" == "plasma-mobile" ]]; then
+     echo "Installing the KDE Plasma Mobile DE..."
+     # Use Yay since plasma mobile is on the AUR
```

Comment:
You're right, any package from the AUR (even if it is packaged in the Chaotic AUR or winesapOS repository) should use `yay`. It's our fail-safe in case all repositories disappeared for whatever reason.

This is kind of implied so you don't need the comment here. I should probably make a note of it in the `CONTRIBUTING.md` guide.